frontend/src/components/secure-message/AccessUserCard.vue

```diff
@@ -234,71 +238,133 @@ export default {
       this.relinkState = !this.relinkState;
     },
     clickActionRelinkButton(userToRelink) {
-      let userSchool = userToRelink.edxUserSchools.find(school => school.mincode === this.mincode);
-      const payload = {params:
-          {
-            userToRelink: userToRelink.edxUserID,
-            mincode: this.mincode,
-            userSchoolID: userSchool.edxUserSchoolID
-          }
-      };
-      ApiService.apiAxios.post(Routes.edx.EXCHANGE_RELINK_USER, payload)
-        .then(()=> {
-          this.setSuccessAlert('User has been removed, email sent with instructions to re-link.');
-        }).catch(error => {
-          this.setFailureAlert('An error occurred while re-linking a user. Please try again later.');
-          console.log(error);
-        }).finally(() => {
-          this.$emit('refresh');
-        });
+      if(this.instituteTypeCode === 'SCHOOL'){
+        let userSchool = userToRelink.edxUserSchools.find(school => school.mincode === this.instituteCode);
+        const payload = {params:
+              {
+                userToRelink: userToRelink.edxUserID,
+                mincode: this.instituteCode,
+                userSchoolID: userSchool.edxUserSchoolID
+              }
+        };
+        ApiService.apiAxios.post(Routes.edx.EXCHANGE_RELINK_USER, payload)
```

Comment:
Can you please use the if/else only for the top payload criteria and remove 268 down?